oshi-core/src/main/java/oshi/driver/unix/freebsd/disk/GeomPartList.java

```diff
@@ -133,7 +133,7 @@ public static Map<String, List<HWPartition>> queryPartitions() {
             }
             // Process last diskstore
             if (!partList.isEmpty()) {
-                partList = partList.stream().sorted(Comparator.comparing(HWPartition::getName))
+                partList = partList.stream().parallel().sorted(Comparator.comparing(HWPartition::getName))
```

Comment:
Doesn't actually do anything here, as the sorting list needs the whole stream anyway.